File: Backend/tecnpoint/src/main/java/com/techsolutions/tecnpoint/DTO/UsuarioLogadoDTO.java
package com.techsolutions.tecnpoint.DTO;

import com.techsolutions.tecnpoint.enums.TipoUsuario;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
@AllArgsConstructor
public class UsuarioLogadoDTO {
    private Long idUsuario;
    private String nome;
    private String email;
    private TipoUsuario tipoUsuario;
}

File: Desktop/TecnPoint/TecnPoint.Modelos/DTO/UsuarioLogadoDTO.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TecnPoint.Modelos.Enum;

namespace TecnPoint.Modelos.DTO
{
    public class UsuarioLogadoDTO
    {
        public long idUsuario { get; set; }
        public string nome { get; set; }
        public string email { get; set; }
        public TipoUsuario tipoUsuario { get; set; }
    }
}
